mac/utils/utils.c

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
I moved mac/utils/utils.c to utils/utils.c. I suppose this file is not only for mac.